packages/nextjs/app/layout.tsx

```diff
@@ -0,0 +1,50 @@
+import "@rainbow-me/rainbowkit/styles.css";
+import { Metadata } from "next";
+import { AppComponent } from "~~/components/AppComponent";
+import "~~/styles/globals.css";
+
+const baseUrl = process.env.NEXT_PUBLIC_VERCEL_URL ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}/` : "/";
```

Comment:
I think it's better without `/` here

Reply:
should i remove `/` in `https://${process.env.NEXT_PUBLIC_VERCEL_URL}/` ??

Reply:
Thought you removed it. Answer is yes :)

Reply:
metadataBase works fine for me with this baseUrl